### PR TITLE
Feat/signup page

### PR DESCRIPTION
**Pull request by frontend-specialisten**
## Description

Add custom layout for sign up (B2C).
Links:
signup: https://bit.ly/3K3xysb
signin: https://bit.ly/34GvVAE

## References

- [555](https://github.com/Energinet-DataHub/geh-aggregations/issues/555)

### CODE REVIEW

**Review by MrDach — COMMENTED** [review]
Focus/blur on input fields causes the interface to bounce due to the border getting removed.

File: libs/dh/shared/assets/src/assets/signup-page/signup-unified.html [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
...X6ny87LE7h9nJj/zPtzLgAyFKrF+T1Q5MZGwQQZYdA+YhSg+BAAf856jXc8wRPZASFDmcWtBKzSbYriO4Sw02v2zPB/pJMsTKVpz3V6x8NswGJ4z2ZjHkhhCRs6h2CJ3jX/2SPzqDOC8nYBnROFC5q1AigGusm+kjHSvGYpFEpOIxKSufWfjWEr4KUl6xq4xUcWvbx4mW1lOLZcHMB8yrhIbLQyrEIdEXxs7h1WQx6hTsHjDO1koufhR6baRfUy+jPKSLExhhvABzTsbj9jqt6ZuYx+5o8UJFrYI15Aa3X5X+RMhCcunGRIqqdLkVaKNICalUVikzTwKQgug/A2CGk0xcknR8NMfyoTj0VovVVfHmnByduHUj6xfmslwmT08oqplvCWUjMg1Pgd/PkU6jDP2OupPCK16TxXhawM3N7KRiiwyaSnEIV1Po2WIIvRVLC06hqz8qKq2d1C/MSBUWWFtVjdQ+q6Oh+RJiTDuz+yZQkTwTOKkkyS251h3ZhXXGCIXA663llBF/dhA1OVaqYce8eEY9IO1m4Mp3Ecl/90jxcE0KEEsEG7TJRXhs1Vu/cSn2MtSFKBktrba5rnmvu94Wmtcx7GEluGUDi4EgjjPGYwr6KK3F+8N68XKfBQIGKAouml6Pw3/DXDERMEipURw7CH3oAY1qNy/2xtrN2BIE63aUs+AXbs9+CBMlSgaELYaysg/8rqaUnjEvfrCkQrAJDKHd5fne+7W9FAlshcZzTRNeEoqQJ2SuAOXq3UDqwB4ZOo6NvnUCJWlbIEWpUiIYMGH0hdkW4TJOC74Eusne2qsIoKIRWQZAoKc63wnkq5UrKvZQY90LG/jBJE/0LBWrPhM6mqjY/2sscW+jUxIQR7i9xg9765x4kyl1F18k32iLNosJtew4VnwPDfw7hgDiv319eE4hF6dtaR+bxbkUbykVWSbhDvg8Cdz6dMk0q9MR0pPh7SjfSeNEyjipXyDNEjptRsY/hVC1vEvh1ozXEwPzuNveg96KDRASoCRgF0o1oU4VuBN/EFTcifKEJgFKqZq/KmN1qPkM7JqTkJ5sicSJ9qEW/ycufJI72CWdxHT+VC7h7CqIdWUVyBv3ML89M5TaimH0mJjXIExrfa8oSeTBlGYyoEsCYjedZGcOT3faTsieIivwtmae8RCarzZWHVkBkyRBJ/vkpeDx1MUHyTh
```
Comment (dzhavat) [inline libs/dh/shared/assets/src/assets/signup-page/signup-unified.html]: suggestion: change the cursor to `pointer` on hover.

**Comment by dzhavat** [comment]
question: It doesn't seem like I can access the sign up page anymore? When I open the link above in a private window, I get redirected to the sign in page instead. Should I do something else?

**Comment by frontend-specialisten** [comment]
> question: It doesn't seem like I can access the sign up page anymore? When I open the link above in a private window, I get redirected to the sign in page instead. Should I do something else?

@dzhavat Seems like the links has changed after Daniel and I have setup all the environments. The links in descriptions are now updated.

**Review by dzhavat — APPROVED** [review]

**Pull request merged**